### DISCUSSION TITLE
v9.5-(1403-6-10)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#86] [Announcements]
using core v1.8.23-mahsa-r3 :
- change default user-agent for ws, http, httpupgrade
- revive fakehost
- fix wireguard mtu bug on some network

fix UI bug when editing h3 splithttp
fix ping issue on large number of config
fix UI bug in warp generator
fix warp ping issue on old device

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v9.5-(1403-6-10)'>v9.5-(1403-6-10)</a>.</em>

**Comment by armanem2** [comment]
توی این ورژن انگاری تنظیمات روتینگ زیادجالب نییت تانصب کردم سروروم پشت کلود بالانیومد وارورهای زیغدی گرفتم close pip
و tls
لازم به ذکره من روی سروردامنه فیلتر وصل میشدم اما الان کامل روی برنامه وصل نمیشه اما روی nikangوصل میشه

**Reply by GFW-knocker** [reply]
احتمالا اختلال لحظه ای نت بوده چون هسته مهسا و نیکا یکسانه 1.8.23r3. 
اگر مشکل پابرجاست نوع کانفیگ و تنظیمات فرگمنت بیان کنید تا تست بشه

**Reply by armanem2** [reply]
فرگمنت روی نت همراه اوله هم روی اتوماتیک تست شد 
هم دستی ست کردم اما کلاانگار اتصال رو برقرارنمیکنه

**Comment by armanem2** [comment]
مهندس جان‌سلام تست شد کلیه کانفیگ های وی لس بنده از کار افتاده 
کانفیگ ها همه از کارافتادن 
وایرگارد وی لساگر نیاز به جزییاتی هست براتون یجا بفرستم همون کانفیگ روی نیکاانجی وصل وپایداره نکته اینه که روی این ورزن قطع میباشد ورژن هشت نه اوکی بود 
![Screenshot_2024-09-01-23-36-54-657_com.MahsaNet.MahsaNG-edit.jpg](https://github.com/user-attachments/assets/5e09eea8-141e-4ff9-83fe-4ab80724bdc7)

![Screenshot_2024-09-01-23-37-06-228_com.MahsaNet.MahsaNG-edit.jpg](https://github.com/user-attachments/assets/766cffbe-dc36-4c9e-a823-7fd3f769417d)

**Reply by armanem2** [reply]
همان کانفیگ در برنامه نیکا انجی در همان گوشی ونت 
![Screenshot_2024-09-01-23-38-06-279_com.MahsaNet.NikaNG-edit.jpg](https://github.com/user-attachments/assets/e1df45b9-d64a-4f49-b4fb-a06b30044bee)

![Screenshot_2024-09-01-23-38-28-715_com.MahsaNet.NikaNG-edit.jpg](https://github.com/user-attachments/assets/2b7cd9cd-8577-4e52-9193-439c5b039d66)

**Reply by armanem2** [reply]
![Screenshot_2024-09-01-23-44-43-107_com.MahsaNet.MahsaNG.jpg](https://github.com/user-attachments/assets/6d8122d1-3768-46fd-a978-ddf88a7e9d63)

![Screenshot_2024-09-01-23-45-11-961_com.MahsaNet.MahsaNG.jpg](https://github.com/user-attachments/assets/ca097dd1-a425-4eb9-beb9-db04aaaa17b6)

**Comment by GFW-knocker** [comment]
@armanem2 

والا ایده ای ندارم 
جز اینکه یبار کامل پاک کنید دوباره نصب کنید
یا قوانین روتینگ پاک کنید شاید با ایپی gcore تداخل داره
چیزی که سرور نشون میده نسخه 9.5 بطور کلی روان داره کار میکنه

**Reply by armanem2** [reply]
کامل پاک کردم دوباره ورژننه نیم ریختم تست کردم موفق به اتصال نشدم هم به کانفیگ های رایگان که بعضی از دوستان میذارن هم اونایی که خودمون میزاریم😜ولیکن بغیر از کانفیگ مخصوص یوتیوب هیچ 
![Screenshot_2024-09-02-20-33-17-500_com MahsaNet MahsaNG](https://github.com/user-attachments/assets/ff45d055-c8df-419b-b5fb-092a1f0b5cc0)
کدوم وصل نشد اما بعد پاک کردم نسخه نه ریختم دوبارهاوکی شد

**Comment by armanem2** [comment]
عرض ادب خدمت تیم خوب برنامه مهسا انحیدوستان من روی گوشی شیائومی پوکو و نسخه نه نیم تست کردم کلیه کانفیگ های شخصی پشت کلود فلر وجی کور وسایر سی دی ان ها وصل نمیشه اما همان کانفیگ ها باپایداری وکیفیت بالا در نیکاانجی ومهسا ورژن نه وهشت کامل وصله لازم به ذکره  ایپی سرور ودامنه سرورهای غوق الذکر کاملا فیلتره

**Comment by MohamadMahdi-Heydari** [comment]
درود و خسته نباشید
**نسخه ویندوز کی اضافه میشه؟**

**Reply by GFW-knocker** [reply]
کلاینت تقریبا رو به اتمامه
فکر کنم تا 1 ماه اینده ریلیز بشه
تا اون موقع از v2rayN-pro استفاده کنید

**Reply by GFW-knocker** [reply]
گروه تلگرام مهسانت برای راهنمایی بیشتر
@mahsa_net

**Comment by cinorid** [comment]
عرض سلام و درود
یک مشکل دیگه هم من بهش برخورد داشتم اینه در نسخه جدید [BPB-Worker-Panel](https://github.com/bia-pain-bache/BPB-Worker-Panel/releases/tag/v2.5.5) که کانفیگهای تروجان اضافه شده. تست اتصال درست کار نمیکنه. دکمه تست با متن قرمز درست کار میکنه اما وقتی روی نوار پایین میزنیم که اتصال فعلی رو چک کنه، برای کانفیگهای تروجان همیشه خطا میده.
![image](https://github.com/user-attachments/assets/d777e3bd-1e9d-47ed-bc89-6c037a701a1d)
به امید آزادی

**Reply by GFW-knocker** [reply]
احتمالا کانفیگش با
gstatic.com/generate_204
مشکل داره.
خود هسته دیفالت gstatic گذاشته ولی قبلا دیفالتش 
google.com/generate_204
بود. ما برا تست پایین از اولی و برای دکمه تست از دومی استفاده میکنیم
هر کدوم ادرس ها رو یک سری کانفیگ مشکل دارن انگار

**Comment by armanem2** [comment]
توسعه دهنده های عزیزبرنامه من بصورت راندوم وطی چندبار هنگامی که لپدیت برنامرو انجام دادم به نسخه نه ونیم روی گوشی های شیائومی تست کردم (۳تاگوشی) کانفیگ های پشت کلود فستلی و... کامل از کار میوفتن واصلا اتصال برقرارنمیشه این رو روی شرایط(دامنه وایپی فیلتر (پشت کلودقلر فستلی وجی کور تست کردم)درکنال تعجب همه این کانفیگ هاباحداکثر سرعت در نسخه نه تاهشت کامل وصلن 
تنها وایرگارد ترکیبی با وارپ بااین نسخه وصل میشد که عوسط هودبرنامه جنریت شده بود راهکاری برای این مشکل جدی ندارید؟

**Reply by GFW-knocker** [reply]
خب طبق فیدبک ما ، نسخه 9.5 کاملا پایداره و 50% یوزرای مهسا دارن استفاده میکنن
هیچ ایده ای برای شما به ذهنم نمیرسه. میتونید یک لاگ کامل برای ما بفرستید در ایمولاتور اجرا کنید یا با NikaNG اخرین نسخه تست کنید که هسته اش با مهسا 9.5 یکسانه.

**Reply by Ligangligang323323** [reply]
非常好用
